sql/telemetry_derived/glam_valid_builds_v1/query.sql

```diff
@@ -0,0 +1,14 @@
+WITH build_id_user_counts AS (
```

Comment:
Do you know how much data this query is using, from one of your test runs?

Reply:
~27GB